docs/quickstart.md

```diff
@@ -0,0 +1,187 @@
+
+As a short overview, a character is represented by a steering agent; it stores its position, orientation, maximum speeds and current velocity. A steering behavior is associated with a steering agent and it calculates a linear and/or angular change in speed from the information that is available. The coder then applies that acceleration in whatever ways is appropriate to the character to change its velocity.
+
+## More information and resources ##
+
+- [Understanding Steering Behaviors](https://gamedevelopment.tutsplus.com/series/understanding-steering-behaviors--gamedev-12732): Breakdowns of various behaviors by Fernando Bevilacqua with graphics and in-depth explanations.
+- [GDX-AI Wiki](https://github.com/libgdx/gdx-ai/wiki/Steering-Behaviors): Descriptions of how LibGDX's AI submodule uses steering behaviors with a few graphics. Since this toolkit uses it for inspiration, there will be some similarities.
+- [RedBlobGames](https://www.redblobgames.com/) - An excellent resources for complex pathfinding like A*, graph theory, and other algorithms that are game-development related. Steering behaviors are not covered, but for anyone looking to study and bulk up on their algorithms, this is a great place.
+
+## Example usage ##
+
+The fastest way to get started is to look at an explained sample class that makes use of the toolkit.
+
+The goal of this class is to represent an agent that chases the player and predicts where the player *will* be, but maintains a distance from him. When its health is low, it will flee from the player directly. The agent will keep facing the player while it is chasing it, but will look where it's going while it is fleeing. Our game will be in 2D, assumed to be a top-down spaceship game.
+
+You can see the demo in action by opening the `demos/QuickStartDemo.tscn` file in Godot. The result is an agent that approaches the player and hovers near him until it is shot enough times, at which point it will try to flee directly.
+
+More details about how the various steering behaviors function can be found in the [Reference](./reference.md) document.
+
+```ruby
+extends KinematicBody2D
+
+
+export var maximum_speed := 450.0 # Maximum possible linear velocity
+export var maximum_acceleration := 50.0 # Maximum change in linear velocity
+export var maximum_rotation_speed := 240 # Maximum rotation velocity represented in degrees
+export var maximum_rotation_accel := 40 # Maximum change in rotation velocity represented in degrees
+export var maximum_health := 100
+export var flee_health_threshold := 20
+
+var velocity := Vector2.ZERO # Represents the current velocity
+var angular_velocity := 0.0 # Represents the current rotational velocity

```

Comment:
These comments are not necessary. The name already says explicitly that this is the angular velocity.